generator/internal/logging_decorator_generator.h

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Google LLC
```

Comment:
I forget: how/where are these generators tested? There's no `logging_decorator_generator_test.cc` file, so is there something else that's invoking and testing this?